WriteVHDLSyntax.py

```diff
@@ -950,6 +1008,12 @@ def writeTBControlSignals(memDict, memInfoDict, initial_proc, final_proc, notfin
                     string_ctrl_signals += ("t_"+mtypeB+"_NENT").ljust(str_len2)+":= (others => '0'); -- (#page)(#bin)\n"
                     string_ctrl_signals += ("  signal "+mem+"_AV_dout_mask").ljust(str_len)+": "
                     string_ctrl_signals += ("t_"+mtypeB+"_MASK").ljust(str_len2)+":= (others => (others => '0')); -- (#page)(#bin)\n"
+                elif split and "TPAR" in mem:
+                    seed = mem.split("_")[1][:-1]
+                    itc = mem.split("_")[1][-1]
+                    for PCGroup in MPARdict[seed]:
+                        if PCGroup[0] == itc:
+                            string_ctrl_signals +="  signal MTPAR_"+seed+PCGroup +"_stream_V_dout: std_logic_vector(75 downto 0) := (others=> '0') ;\n"
```

Comment:
@jasonfan393 Can we avoid the hard-coded `75`?
```suggestion
                            string_ctrl_signals +="  signal MTPAR_"+seed+PCGroup +"_stream_V_dout: std_logic_vector(" + str(memMod.bxbitwidth + memMod.bitwidth - 1) + " downto 0) := (others=> '0') ;\n"
```

Or maybe the type should be defined in memUtil_pkg.vhd?